tools/envoy_headersplit/BUILD

```diff
@@ -0,0 +1,46 @@
+load("@rules_python//python:defs.bzl", "py_test")
+load("@headersplit_pip3//:requirements.bzl", "requirement")
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_package()
+
+py_test(
+    name = "headersplit_test",
+    srcs = [
+        "headersplit.py",
```

Comment:
Shouldn't you depend on `headersplit_pip3` and then not bring in the file directly?